redi/utils/GetEmrData.py

```diff
@@ -161,7 +166,24 @@ def get_emr_data(conf_dir, connection_details):
     :conf_dir configuration directory name
     :connection_details EmrFileAccessDetails object
     """
-    raw_txt_file = os.path.join(conf_dir, 'raw.txt')
-
-    # download csv file
-    download_file(raw_txt_file, connection_details)
+    # enable backwards comparability with older config repos
+    # try reading emr_data_file as a dict first
+    try:
+        connection_details.download_file = ast.literal_eval(connection_details.download_file)
```

Comment:
I would move the logic about parsing "file_a" or "[file_a, file_b]" with  ast.literal_eval(connection_details.download_file) to the EmrFileAccessDetails class __init__() method. The advantage of this approach is that users of the class can expect the   `connection_details.download_file` to always be a list containing one or more entries. This will also simplify the code so we would not need to catch the exception "except ValueError" on line https://github.com/ctsit/redi/pull/167/files#diff-fc434789abf3ec875ce19da895ef550aR184